graylog2-web-interface/src/views/bindings.jsx

```diff
@@ -249,7 +249,7 @@ export default {
     {
       type: 'create-extractor',
       title: 'Create extractor',
-      isEnabled: (({ type }) => (type.type === 'string' && !type.isDecorated()): ActionHandlerCondition),
+      isEnabled: (({ type, contexts }) => (!!contexts.message && type.type === 'string' && !type.isDecorated()): ActionHandlerCondition),
```

Comment:
Interesting learning: I had to implement the boolean casting, otherwise the function would not return `false` but `undefined`. I thought it is enough to do the chaining with `&&`.

Reply:
Returning `undefined` here is okay, as falsyness is considered sufficient.